### DISCUSSION TITLE
OSCS 开源软件供应链安全社区

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#81] [2️⃣ 聊项目]
看到一个开源漏洞检测的项目，整个过程和方法，挺符合我们想做的 OpenCerfified 项目的想法的~

网站是：https://www.oscs1024.com/

以我们的 hypertrons-crx 项目为例，在他的搜索框中输入后，就可以看到项目的结果了。

![image](https://user-images.githubusercontent.com/15010826/172321946-e6a6d80d-6622-4b27-a141-505a32562585.png)

结果连接信息：https://www.oscs1024.com/cd/1532281285341851648

然后可以加入他们，以及修复后，给予徽章，整个产品的逻辑还是非常值得学习的~

背后也有开源项目的支持：https://github.com/murphysecurity/murphysec

大家可以看看学习下~ @frank-zsy @TieWay59 @tyn1998 @gymgym1212

**Comment by will-ww** [comment]
下周找个时间，邀请 @TieWay59 给我们仔细分享下吧~ :-)

**Reply by TieWay59** [reply]
收到~
